frontend/packages/pipelines-plugin/src/const.ts

```diff
@@ -4,3 +4,16 @@ export const CLUSTER_PIPELINE_NS = 'openshift';
 export const PIPELINE_RUNTIME_LABEL = 'pipeline.openshift.io/runtime';
 export const PIPELINE_RUNTIME_VERSION_LABEL = 'pipeline.openshift.io/runtime-version';
 export const PIPELINE_STRATEGY_LABEL = 'pipeline.openshift.io/strategy';
+
+export enum runStatus {
+  Succeeded = 'Succeeded',
+  Failed = 'Failed',
+  Running = 'Running',
+  'In Progress' = 'In Progress',
+  FailedToStart = 'FailedToStart',
+  PipelineNotStarted = 'PipelineNotStarted',
+  Skipped = 'Skipped',
+  Cancelled = 'Cancelled',
+  Pending = 'Pending',
+  Idle = 'Idle',
+}
```

Comment:
Since this is not really a constant, I recommend to move this and rename it also. Before we continue let see if @karthikjeeyar and/or @vikram-raj are fine with this recommendation:

The `pipelines-plugin` has the tekton model types in `packages/pipelines-plugin/src/types/*.ts`.

But because this is not a type defined by tekton, I recommend making this clear by:

1. rename it to `ComputedStatus` (uppercase C)
2. move it into `packages/pipelines-plugin/src/types/computedStatus.ts` (lowercase c, similar to other files)
3. Export it in `packages/pipelines-plugin/src/types/index.ts`
4. Add this comment:

```ts
// The enum values need to match the dynamic-plugin `Status` `status` prop.
// Translation (`title` prop) is computed via separate reducers.
export enum ComputedStatus {
```

Reply:
+1 move it `computedStatus.ts` as it is not const.